README.md

```diff
@@ -55,6 +55,14 @@ Anything you type in your terminal will appear in your dash. To exit, simply typ
 exit
 ```
 
+## Record command output
+
+To record the output of a command in the Dashcam app (In this example the `ping 1.1.1.1` command ), use the following command
+
+```
+ping 1.1.1.1 | dashcam x
```

Comment:
Can we call this command `pipe` instead of `x`

Reply:
It's done